mattermost/mattermost-webapp
# [MM-42357] Change mention search to use new mention parameter

#### Summary

Changes the mention search to use the new `search_mentions` parameter. This allows group mentions to appear in the recent mentions panel. It currently won't work until the corresponding server PR is merged. 



#### Ticket Link

[Jira ticket](https://mattermost.atlassian.net/browse/MM-42357)

#### Related Pull Requests

- Has server changes: https://github.com/mattermost/mattermost-server/pull/21659

Test steps:
- Open the recent mentions panel and verify it is now showing group mentions in addition to username mentions, first name mentions if the setting is on, and posts containing the user's mention keys.
- Note that the cloud server is not building with the server branch, so you will have to test locally or wait for the server PR to be merged.

#### Release Note

```release-note
Added group mentions to the recent mentions.
```
